.flake8

```diff
@@ -1,33 +1,7 @@
 [flake8]
-exclude=
-    .git,
-    dev-requirements,
-    venv,
```

Comment:
Should keep these includes in there (a) to speed things up, and (b) so that we don't style-check things installed into virtual environments

Reply:
done